api/snakeyaml-engine-kmp.api

```diff
@@ -844,6 +844,9 @@ public final class it/krzeminski/snakeyaml/engine/kmp/exceptions/YamlVersionExce
 	public fun <init> (Lit/krzeminski/snakeyaml/engine/kmp/common/SpecVersion;)V
 }
 
+public abstract interface annotation class it/krzeminski/snakeyaml/engine/kmp/internal/CopyDsl : java/lang/annotation/Annotation {
+}
```

Comment:
It's public to be able to resolve it in the processor's unit tests. I added a docstring saying that it's visible only for testing. It's also not really usable by the users of the lib, they cannot do anything useful with this annotation as it's used only in build time (unless repurposed by the user).

Reply:
Ah, now it's clear:

> The `internal` visibility modifier means that the member is visible within the same module. More specifically, a module is a set of Kotlin files compiled together

So the tests need the annotation to be public because the annotation is compiled in a different compiler run than the code in the test (where we call the compiler manually). I'll move the annotation to the file passed to the compiler manually, to reproduce the case we have in the lib's code.